awstesting/integration/customizations/s3/s3crypto/s3_crypto.feature

```diff
@@ -0,0 +1,18 @@
+# language: en
+@s3 @client
```

Comment:
`@s3` probably should be something like `@s3encclient` to be specific to these integration tests. `@client` is generally reserved for the generated service client too.